Modules/mmapmodule.c

```diff
@@ -1951,6 +1952,7 @@ new_mmap_object(PyTypeObject *type, PyObject *args, PyObject *kwdict)
         PyErr_SetFromErrno(PyExc_OSError);
         return NULL;
     }
+    _PyAnnotateMemoryMap(m_obj->data, map_size, "cpython:new_mmap_object");
```

Comment:
```suggestion
    _PyAnnotateMemoryMap(m_obj->data, map_size, "cpython:mmap");
```
(If we allow users to set an annotation: this could be `cpython:mmap:xxx`?)

Reply:
I don't think `_PyAnnotateMemoryMap` should be used in this case at all:

1. This function creates a memory mapping on behalf of the caller, not Python itself. So it's not really correct to attribute the memory region to Python.
2. You can only name anonymous memory areas, but this region may not be anonymous.

Reply:
I think it's better to annotate this as allocated with the Python `mmap` module, rather than by something else (within CPython, or another library).

If the mmap is not anonymous, the annotation fails. Good.
If in the future, the kernel somehow allows annotating non-anonymous areas, this annotation would also work.